Signum.React/Scripts/Lines/AutoCompleteConfig.tsx

```diff
@@ -188,21 +186,32 @@ export class FindOptionsAutocompleteConfig implements AutocompleteConfig<ResultR
         filters = [...defaultFilters, ...filters];
     }
 
-    if (FindOptionsAutocompleteConfig.liteKeyRegEx.test(subStr)) {
-      const lite = parseLite(subStr);
-      const tis = getTypeInfos(qd.columns["Entity"].type);
-      const ti = tis.singleOrNull(ti => ti.name == lite.EntityType);
+    const lites = parseLiteList(subStr);
+    if (lites.length > 0) {
+      const type = qd.columns["Entity"].type;
+      const tis = getTypeInfos(type);
+      const ti = tis.singleOrNull(ti => ti.name == lites[0].EntityType);

```

Comment:
I don't understand this complexity. Why not filter all the `lites` by having one of the types in `tis` and then add a `Entity` `EqualsTo` / `IsIn` with the list.

Reply:
done 👍